rdflib/parser.py

```diff
@@ -189,6 +189,9 @@ def create_input_source(
     auto_close = False  # make sure we close all file handles we open
     if location is not None:
         # Fix for Windows problem https://github.com/RDFLib/rdflib/issues/145
```

Comment:
Please keep comments with the code they're commenting on.

Reply:
Fixed this.